CHANGES.md

```diff
@@ -22,6 +22,8 @@ Features
 * [#547](https://github.com/java-native-access/jna/pull/547): Added `GetSystemTimes` to `com.sun.jna.platform.win32.Kernel32` - [@dbwiddis](https://github.com/dbwiddis).
 * [#548](https://github.com/java-native-access/jna/pull/548): Return 64-bit unsigned integer from `com.sun.jna.platform.win32.WinBase.FILETIME` - [@dbwiddis](https://github.com/dbwiddis).
 * [#524](https://github.com/java-native-access/jna/pull/524): Added IShellFolder interface plus necessary utility functions to Windows platform, and a sample for enumerating objects in My Computer - [@lwahonen](https://github.com/lwahonen).
+* [#552](https://github.com/java-native-access/jna/pull/552): Added `Module32First` and `Module32Next` to Kernel32 (and helper to Kernel32Util) and `MODULEENTRY32` structure to Tlhelp32 - [@mlfreeman2](https://github.com/mlfreeman2).
```

Comment:
Please expand `Kernel32` into `com.sun...` like the other ones, not all of this is Windows. Thx.